src/main/kotlin/com/maitre/menuservice/adapter/group/in/GroupRouter.kt

```diff
@@ -1,17 +1,21 @@
 package com.maitre.menuservice.adapter.group.`in`
 
 import com.maitre.menuservice.adapter.group.`in`.create.CreateGroupHandler
+import com.maitre.menuservice.adapter.group.`in`.get.GetGroupHandler
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 import org.springframework.web.reactive.function.server.router
 
 @Configuration
 class GroupRouter(
         private val createGroupHandler: CreateGroupHandler,
+        private val getGroupHandler: GetGroupHandler,
 ) {
 
     @Bean
     fun routerGroup() = router {
         POST("/group", createGroupHandler::execute)
+        GET("/group", getGroupHandler::execute)
     }
```

Comment:
For convention, let's equalize the names:

Class MenuRouter, method menuRoutes
Class ProductRouter, method productRoutes
Class **GroupRouter**, method **groupRoutes**

Shall we?

Reply:
done